doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md

```diff
@@ -59,7 +59,7 @@ Settings for default output filters shipped with Pimcore.
 
 
 ## Web Service API
-Settings fpr Pimcore web service API. 
+Settings for Pimcore web service API. 
```

Comment:
this point should go completely. please see https://github.com/pimcore/pimcore/pull/7279

Reply:
didn't know that. Just started learning pim